WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1478,6 +1478,23 @@ ContactListPaginationChild          = display:inline-block;
 ContactListPaginationLink           = 
 ContactListPaginationLinkIcon       = height:30px; width:30px;
 
+# Contact Search
+
+ContactSearchWidget                    = width:500px;
+ContactSearchResultsContainer          = margin-top:@SkinWrapperPadding@; @PanelBorder@
+ContactSearchListColHeader             = display:none;
```

Comment:
Why have the header elements in DOM . If possible at widget level we should have a configuration to not have any header.